app/controllers/api/v1/posts_controller.rb

```diff
@@ -1,4 +1,4 @@
-class PostsController < ApplicationController
+class Api::V1::PostsController < Api::V1::BaseController
```

Comment:
```suggestion
module Api
  module V1
    class PostsController < Api::V1::BaseController
```

Так же для остальных

Reply:
Это так обязательно делать, в смысле, общепринято? У нас на вебинаре делали, как я сделал. В чем разница?